src/Widgets/ThumbnailGalleryWidget/ThumbnailGalleryWidgetComponent.js

```diff
@@ -47,15 +46,71 @@ Scrivito.provideComponent("ThumbnailGalleryWidget", ({ widget }) => {
             />
           ))}
         </div>
-        <ReactBnbGallery
+        <Modal
           show={lightboxIsOpen}
-          backgroundColor="rgba(22,22,22,.9)"
-          zIndex={1000000} // same value as .cookie-box
-          activePhotoIndex={currentImage}
-          photos={lightboxImages}
-          onClose={closeLightbox}
-          wrap={false}
-        />
+          onHide={closeLightbox}
+          centered
+          size="xl"
+          restoreFocus={false}
+          className="thumbnail-gallery-widget--modal"
+        >
+          <button
+            type="button"
+            className="slick-slide-close-button"
+            aria-label="Close"
+            onClick={closeLightbox}
+          >
+            <svg
+              viewBox="0 0 24 24"
+              style={{
+                height: "2em",
+                width: "2em",
+                display: "block",
+                fill: "#fff",
+              }}
+            >
+              <path
+                d="M23.25 24c-.19 0-.38-.07-.53-.22L12 13.06 1.28 23.78c-.29.29-.77.29-1.06 0s-.29-.77 0-1.06L10.94 12 .22 1.28C-.07.99-.07.51.22.22s.77-.29 1.06 0L12 10.94 22.72.22c.29-.29.77-.29 1.06 0s.29.77 0 1.06L13.06 12l10.72 10.72c.29.29.29.77 0 1.06-.15.15-.34.22-.53.22"
+                fillRule="evenodd"
+              />
+            </svg>
+          </button>
+          <Slider
+            dots
+            dotsClass="slick-dots slick-thumb"
+            infinite
+            speed={500}
```

Comment:
* [ ] FYI I think `500` is the default, so we could drop this. Alternatively, the old implementation behaved like `0`, so that could be an alternative.